content/en/docs/tasks/debug/debug-cluster/flow-control.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Troubleshooting APIServer flow control issues
+content_type: task
+---
+
+<!-- overview -->
+
+API Priority and Fairness controls the behavior of the Kubernetes API server in
+an overload situation. You can find more information about it in the
+[concept doc](/docs/concepts/cluster-administration/flow-control.md)
+
+## {{% heading "prerequisites" %}}
+
+* Kubernetes cluster is installed
+* `kubectl` is configured to communicate with the cluster
+
```

Comment:
I imagine you also need API Priority and Fairness enabled?

Reply:
```suggestion
- You must have the `APIPriorityAndFairness`
  [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
  enabled in your cluster (this feature is enabled by default for Kubernetes versions
  1.20 through to {{< skew currentVersion >}}, but you can explicitly disable it).

```

Reply:
No longer need this part since it's not a troubleshooting doc any more.